### PR TITLE
Make number of vnodes (num_tokens) configurable

### PR DESCRIPTION
**Pull request by a1exsh**
As a test, only for version 2 for now.

### CODE REVIEW

**Comment by a1exsh** [comment]
>
> Tried to avoid "Number of vnodes per node"?
>
You get it. ;-)

File: README.rst [hunk 1]
```diff
@@ -54,6 +54,7 @@ Available options are::
 
     --cluster-name	Not actually an option, you must specify the name of a cluster to create
     --cluster-size	Number of nodes to create per AWS region.  Default: 3
+    --num-tokens	Number of vnodes per instance (see num_tokens).  Default: 256
```
Comment (lmineiro) [inline README.rst]: The previous statement says exactly "Number of nodes". We should be consistent.

Comment (a1exsh) [inline README.rst]: Fair.  Reworded to make it clearer.

**Comment by lmineiro** [comment]
:+1:

**Comment by a1exsh** [comment]
:+1:

**Pull request merged**